.github/workflows/provision.yml

```diff
@@ -0,0 +1,128 @@
+name: Provision environment
+run-name: Provision ${{ github.event.inputs.environment }}
+on:
+  workflow_dispatch:
+    inputs:
+      environment:
+        type: choice
+        description: Machine to provision
+        default: qa
+        required: true
+        options:
+          - development
+          - staging
+          - qa
+          - production
+      tag:
+        type: choice
+        description: Select group tag you want to execute
+        default: all
+        options:
+          - all
+          - backups
+          - checks
+          - application
+          - tools
+          - docker
+          - deployment
+          - users
+          - crontab
+          - mongodb
+          - data
+          - swap
+          - ufw
+          - fail2ban
+          - decrypt
+          - swarm
+          - elasticsearch
+          - traefik
+      debug:
+        type: boolean
+        description: Open SSH session to the runner after deployment
+        default: false
+jobs:
+  provision:
+    environment: ${{ github.event.inputs.environment }}
+    runs-on: ubuntu-20.04
+    outputs:
+      outcome: ${{ steps.deploy.outcome }}
+    timeout-minutes: 60
+    steps:
+      - name: Clone country config resource package
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+          path: './${{ github.event.repository.name }}'
+
+      - name: Set variables for ansible in production environments
+        id: ansible-variables
+        run: |
+          JSON_WITH_NEWLINES=$(cat<<EOF
+            ${{ toJSON(env) }}
+          EOF)
+          JSON_WITHOUT_NEWLINES=$(echo $JSON_WITH_NEWLINES | jq -R -c .)
+          echo "EXTRA_VARS=$JSON_WITHOUT_NEWLINES" >> $GITHUB_OUTPUT
+        env:
+          encrypted_disk_size: ${{ vars.DISK_SPACE }}
+          disk_encryption_key: ${{ secrets.ENCRYPTION_KEY }}
+          dockerhub_username: ${{ secrets.DOCKER_USERNAME }}
+          dockerhub_password: ${{ secrets.DOCKER_TOKEN }}
+          mongodb_admin_username: ${{ secrets.MONGODB_ADMIN_USER }}
+          mongodb_admin_password: ${{ secrets.MONGODB_ADMIN_PASSWORD }}
+          backup_encryption_passphrase: ${{ secrets.BACKUP_ENCRYPTION_PASSPHRASE }}
```

Comment:
**Breaking change:**
- New environment variable `BACKUP_ENCRYPTION_PASSPHRASE` needs to be defined in Github Secret. This variable is an encryption key used to encrypt backup archives.